pkg/replica/backup.go

```diff
@@ -59,15 +59,34 @@ func (rr *RestoreStatus) UpdateRestoreStatus(snapshot string, rp int, re error)
 	rr.SnapshotName = snapshot
 	rr.Progress = rp
 	if re != nil {
-		rr.Error = re.Error()
+		if rr.Error != "" {
+			rr.Error = fmt.Sprintf("%v: %v", re.Error(), rr.Error)
```

Comment:
Notice that `rr.Error` is type `string`. You mean `rr.Error = fmt.Errorf("%v: %v", re.Error(), rr.Error).Error()`? I don't think this can improve the readibility.